### ISSUE TITLE
kubens with fzf prints escape characters in the notice after choosing namespace 

### ISSUE THREAD

**Issue by alexdyas** [ahmetb/kubectx#258]
```
% kubens
# choose kube-system namespace using fzf functionality
✔ Active namespace is "\x1b[32mkube-system\x1b[0m".
```
- Go version
- v0.9.1

I don't see this behavior with kubectx

Cheers.

**Comment by ahmetb** [comment]
#228 tracks this.

**Comment by eshepelyuk** [comment]
It's already fixed in 0.9.3
